src/rgw/rgw_bucket.cc

```diff
@@ -3276,6 +3277,13 @@ int RGWBucketCtl::read_bucket_info(const rgw_bucket& bucket,
                                    const BucketInstance::GetParams& params,
                                    RGWObjVersionTracker *ep_objv_tracker)
 {
+  req_state* s = info->s;
+  #ifdef WITH_JAEGER
+    span_structure ss;
```

Comment:
can you rename `s/ss/span_stack/` or something relevant 
self reminder: /TBD

Reply:
"@suab321321 have you considered checking is tracer is initialized using an

create new tracer}
condition?"
Not till now, once I finish my optional_span thing first then I will start to implementing these new features, it will take another two days to make changes

"@suab321321 have you considered checking is tracer is initialized using an

create new tracer}
condition?" same goes for this